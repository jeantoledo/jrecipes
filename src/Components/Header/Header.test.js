import React from 'react';
import { render } from '@testing-library/react';
import Header from './index';
import renderer from 'react-test-renderer';

test('renders app title', () => {
  const { getByText } = render(<Header />);
  const linkElement = getByText(/Recipes/i);
  expect(linkElement).toBeInTheDocument();
});

test('renders header correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

