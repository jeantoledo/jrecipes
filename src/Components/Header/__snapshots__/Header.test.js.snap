// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header correctly 1`] = `
<div
  className="makeStyles-root-76"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h6
        className="MuiTypography-root makeStyles-title-77 MuiTypography-h6"
      >
        Recipes
      </h6>
    </div>
  </header>
</div>
`;
